spec/controllers/students_controller_spec.rb

```diff
@@ -41,6 +59,61 @@
       end
     end
 
+    describe '#bulk_modify' do
```

Comment:
I realized bulk_modify is actually entirely untested, so in addition to tests for bulk setting sections, I added tests for bulk setting everything else too.